### ISSUE TITLE
ZFS (0.6.3) auto mount on startup in Debian

### ISSUE THREAD

**Issue by ksb4ever** [zfsonlinux/zfs#2498]
Hi!
When in /etc/default/zfs ZFS_MOUNT is set to 'yes', no pools are available (and not mounted on startup, of course).
```
# zfs list
no datasets available
# zpool list
no pools available
# zpool status
no pools available
```
It is possible to mount zfs pool only after zpool import id/name.
If ZFS_MOUNT is set to 'no', zpools are available to mount.

**Comment by behlendorf** [comment]
Closing as out of date due to the overhaul of the init scripts.  Please open a new issue if this remains a problem.
